GitCommands/Settings/AppSettings.cs

```diff
@@ -746,6 +746,12 @@ public static bool DontConfirmResolveConflicts
             set { SetBool("DontConfirmResolveConflicts", value); }
         }
 
+        public static bool DontConfirmUndoLastCommit
```

Comment:
Can we please invert it? It is much harder for a human brain to embrace the negative.

Reply:
I named it following the same pattern of all the "don't confirm" settings...
Do you really think I should do differently?

Reply:
It reads awkward.
I haven't had a chance to look in the code, do we have many of those "DontXxx"?

Reply:
8 "DontConfirmXxx" that are settings to avoid the confirmation popups and 2 or 3 more "DontYyy" for other things....

Reply:
Righto. I suppose we'll keep them as-is and refactor later.
Thanks